### DISCUSSION TITLE
Frigate on HA - What would cause Frigate to restart each  / most nights at 3AM CST

### DISCUSSION THREAD

**Discussion by MickPBduece** [blakeblackshear/frigate#12208] [General]
I am a inquisitive hobbiest with an HA setup that I really like but need a little help.  For some time,  Frigate seems to recycle at 3AM.  All the sensors go offline and then comeback.  I also haven't quite figured out the parked car thing for Frigate. So, I get an alert at 3 AM.

Any thoughts on what I've done or why  that Frigate stops and starts at 3 AM ?

Frigate                   14.0.3 Beta w/ Full Access
running via 
Core                      2024.6.4
Supervisor             2024.06.2
Operating System 12.4

**Comment by hawkeye217** [comment]
Are you sure it's frigate itself that's restarting? Maybe your camera is restarting at 3am every night and causing frigate to reconnect to it? Many cameras have a "auto-reboot" function for maintenance.
